client/app/bundles/course/assessment/question/programming/components/sections/SubmitWarningDialog.tsx

```diff
@@ -0,0 +1,28 @@
+import ConfirmationDialog from 'lib/components/core/dialogs/ConfirmationDialog';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import translations from '../../../../translations';
+
+interface SubmitWarningDialogProps {
+  open: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+}
+
+const SubmitWarningDialog = (props: SubmitWarningDialogProps): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <ConfirmationDialog
```

Comment:
This is already deprecated?

Reply:
Ported as is with minor modifications from the old form.

Reply:
I have since migrated this component to the new `Prompt`.